src/scripts/salasana.js

```diff
@@ -0,0 +1,46 @@
+import {startProcess, stopProcess} from '/shared/scripts/progressbar.js';
+import {eventHandled} from '/shared/scripts/uiUtils.js';
+import {showSnackbar} from '/shared/scripts/snackbar.js';
+import {changePassword} from "/scripts/callRest.js";
+
+window.initializeVaihto = function () {
+  console.log('Initializing Salasana');
+  addFormHandlingEventListeners();
+};
+
+function addFormHandlingEventListeners() {
+  const changeForm = document.getElementById('change');
+  changeForm.addEventListener('submit', changeEvent);
+}
+
+async function changeEvent(event) {
+  console.log('Change submit event');
+  eventHandled(event);
+
+  const form = document.getElementById('change');
+  const formData = new FormData(form);
+  const submitButton = document.getElementById('form-change-password-submit-button');
+
+  startProcess();
+  try {
+    submitButton.setAttribute('disabled', true);
+    const {message} = await changePassword({curPass: formData.get('currentPassword'), newPass: formData.get('newPassword'), newPassVerify: formData.get('newPasswordVerify')});
```

Comment:
Is it necessary to use names for object attributes such as **curPass**, **newPass** and so on instead of just currentPassword, newPassword and newPasswordVerified?

Reply:
Nope. Ill change them